local-volume/README.md

```diff
@@ -102,30 +102,73 @@ $ ALLOW_PRIVILEGED=true LOG_LEVEL=5 FEATURE_GATES=PersistentLocalVolumes=true ha
 
 ### Step 2: Creating local persistent volumes
 
-#### Option 1: Bootstrapping the external static provisioner
+#### Option 1: Deploying the external static local provisioner 
 
-This is optional, only for automated creation and cleanup of local volumes. See
-[bootstrapper/](./bootstrapper) and [provisioner/](./provisioner) for details and
-sample configuration files.
+This option uses **helm** templating to generate the provisioner's DaemonSet and ConfigMap specs.
+The generated specs can be further customized as needed (usually not necessary), and then deployed using kubectl.
+
+To install helm's client interface, run the following two command:
 
-1. Create an admin account with cluster admin priviledge:
 ``` console
-$ kubectl create -f bootstrapper/deployment/kubernetes/<version>/admin-account.yaml
+export HELM_URL=http://storage.googleapis.com/kubernetes-helm/helm-v2.7.2-linux-amd64.tar.gz
+curl "$HELM_URL" | sudo tar --strip-components 1 -C /usr/bin linux-amd64/helm -zxf -
+```
+Provisioner's spec generation process has been tested with helm version 2.7.2.
+
+**helm template** command uses 3 sources of information:
+1. Provisioner's chart template located at helm/provisioner/templates/provisioner.yaml
+2. Provisioner's default values.yaml which contains variables used for rendering a template.
+3. (Optional) User's customized values.yaml as a part of helm template command. The default values.yaml can be overrideen
+   by creating and specifying your own values.yaml.
+
+Below is the default values.yaml. All the parameters can be modified, but none should be removed.
+The only exception is the list of storage classes and respective mounting points. At least one storage class
+config is required.
+```
```

Comment:
Maybe we shouldn't put the whole defaults in the README, just point to it.  Otherwise, we have to update this everytime something changes.

Reply:
done